.travis.yml

```diff
@@ -1,17 +1,35 @@
 language: python
-python:
-  - "2.6"
-  - "2.7"
-  - "3.2"
-  - "3.3"
-  - "3.4"
-  - "pypy"
-script: make test
+script: tox
```

Comment:
Agree, assuming there are no hidden regrets, this will make our local tox runs much more symmetric with travisci. Yay.